app/src/androidTest/java/org/systers/mentorship/view/fragments/ChangePasswordFragmentTest.kt

```diff
@@ -0,0 +1,120 @@
+package org.systers.mentorship.view.fragments
```

Comment:
Looks good to me. Please follow the commit message guidelines. Thank you for your contribution 🎉